tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1309,50 +1331,59 @@ finish_routine(void)
   }
 }
 
+/**
+   \brief Return current loop id and mark it as vectorizable.
+
+   No dependency checking currently means to mark a loop id metadata with
+   !"llvm.loop.vectorize.enable", and to mark memory instructions with
+   !llvm.mem.parallel_loop_access referring to the loop id. This function
```

Comment:
Nit: This comment has to be updated. We do not generate llvm.mem.parallel_loop_access now.